### PR TITLE
Add CODEOWNERS file to .github directory

### PR DESCRIPTION
**Pull request by davegill**
TYPE: new feature

KEYWORDS: CODEOWNERS

SOURCE: internal

DESCRIPTION OF CHANGES:
This is the start of a list for shared and individual responsibility for files and directories. When a PR is issued that modifies one of the known files or files within a known directory (and required reviews are enabled), then an automatic request for review is issued.

More info:
```
https://help.github.com/articles/about-codeowners/
```

LIST OF MODIFIED FILES:
M	     .github/CODEOWNERS

TESTS CONDUCTED:
 - [ ] Going to try out some fake PRs to see what happens.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+/dyn_nmm				@llpcarson
+
+#	Registry
+
+/Registry				            @davegill
+/Registry/Registry.NMM			@llpcarson
+/Registry/registry.tracker	@llpcarson
+/Registry/registry.tornado	@llpcarson
+
+/Registry/EM_CHEM			      @ravanah @kayeekayee @stacywalters
```
Comment (davegill) [inline .github/CODEOWNERS]: The file name is `Registry.EM_CHEM`, not `EM_CHEM`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+/chem					@ravanah @kayeekayee @stacywalters
+
+#	Physics
+
+/phys					@dudhia @weiwangncar @llpcarson
+/phys/module_mp_thompson.F		@gthompsnWRF
+
+#	Data Assimilation
+
+/wrftladj				@liujake
```
Comment (davegill) [inline .github/CODEOWNERS]: @jjguerrette 
JJ,
Should you be also a code owner for the wrftladj and var directories?

Comment (jjguerrette) [reply .github/CODEOWNERS]: I think @liujake would like that in the future, but I am not sure if we are ready for that transition.  Let's ask when he is back in town.

**Comment by davegill** [comment]
Better PR is #658

**Pull request closed without merging**